src/main/resources/timeline/article/index.scala.html

```diff
@@ -1,20 +1,39 @@
 @import com.gu.contentatom.renderer.utils.GuardianDateFormatter
-@(atom: com.gu.contentatom.thrift.Atom
-, data: com.gu.contentatom.thrift.atom.timeline.TimelineAtom
+@import com.gu.contentatom.thrift.atom.timeline.TimelineItem
+@(atom: com.gu.contentatom.thrift.Atom,
+        data: com.gu.contentatom.thrift.atom.timeline.TimelineAtom
 )
 
+@renderFormattedDate(item: TimelineItem, format: String) = @{
+  val formatter = GuardianDateFormatter.customFormat(format)
+
+  item.toDate match {
+    case Some(date) => {
+      val beginRange = GuardianDateFormatter.toMachineValue(item.date).format(formatter)
+      val endRange = GuardianDateFormatter.toMachineValue(date).format(formatter)
+      s"$beginRange&ndash;$endRange"
+    }
+    case None => GuardianDateFormatter.toMachineValue(item.date).format(formatter)
+  }
+}
+
 @fragments.atoms.html.snippet(
   atom.id,
   className = "timeline",
   label = data.typeLabel.getOrElse("Timeline"),
   headline = atom.title.getOrElse("")
 ){
+  @for(description <- data.description) {
+    <div class="atom-snippet__description explainer-snippet__description">@Html(description)</div>
```

Comment:
Aim for `atom--<atomCategory>__<component>`, e.g. `atom--snippet__description` (and update the CSS accordingly)